src/cli/repo/repo.go

```diff
@@ -55,6 +58,8 @@ func AddCommands(cmd *cobra.Command) {
 		addRepoTo(initCmd)
 		addRepoTo(connectCmd)
 	}
+
+	addS3Commands(updateCmd)
```

Comment:
this will create the command chain `corso update-passphrase s3`. Do we really need the `s3` part?

Reply:
Now when I think of it, https://github.com/alcionai/corso/pull/4438 won't be required in that case.